scripts/cf-check.sh

```diff
@@ -3,15 +3,15 @@ set -e
 if command -v cf /dev/null 2>&1; then
     echo The command cf is available
 else
-
     apt-get update
     apt-get install wget gnupg2 apt-transport-https
 
-    wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
-
-    echo "deb https://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
-
-    apt-get update
-    apt-get install cf7-cli
+    NEXUS_ARCHIVE="cf7-cli_7.7.13_linux_x86-64.tgz"
+    NEXUS_URL="https://tdp-nexus.dev.raftlabs.tech/repository/tdp-bin/cloudfoundry-cli/$NEXUS_ARCHIVE"
```

Comment:
does this help keep our pipelines stable during the annual cf cli issue we've seen in recent years? @andrew-jameson

Reply:
tagging one more time for visibility 😄 ⬆️  @andrew-jameson @jtimpe @raftmsohani @elipe17

Reply:
Yes indeed it should save us from them not rolling their certs in a timely manner.